packages/application/src/submission.tsx

```diff
@@ -176,10 +177,25 @@ export class SubmissionHandler {
           dialogTitle = 'Job submission to ' + runtime_config + ' succeeded';
           dialogBody = (
             <p>
-              Check the status of your run at{' '}
-              <a href={data.url} target="_blank" rel="noopener noreferrer">
-                Run Details
+              Check the status of your experiment at{' '}
+              <a
+                href={data['experiment-url']}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Run Details.
               </a>
+              <br />
+              The experiment results and outputs are in{' '}
+              <a
+                href={data['object-storage-url']}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                object storage
+              </a>{' '}
+              <br /> and located in the {data['object-storage-path']} working
```

Comment:
I feel like the English here is a bit off:
`The experiment results and outputs are in object storage and located in the /cos/dir working directory.`

I think grammatically this would be better (again note I'm not a fan of the work `experiment` in the dialog):
`The results and outputs are located in the /cos/dir directory in object storage.`